test/Microsoft.Azure.WebJobs.Host.EndToEndTests/AsyncChainEndToEndTests.cs

```diff
@@ -436,10 +436,10 @@ public void FunctionFailures_LogsExpectedMessage()
             [Blob(ContainerName + "/" + NonWebJobsBlobName, FileAccess.Write)] Stream nonSdkBlob,
             CancellationToken token)
         {
-            queueMessages.Add(" works");
-
             byte[] messageBytes = Encoding.UTF8.GetBytes("async");
             await nonSdkBlob.WriteAsync(messageBytes, 0, messageBytes.Length);
+
+            queueMessages.Add(" works");
         }
```

Comment:
Well... that didn't seem to work. Thinking through this again. Maybe WriteAsync isn't guaranteed to be completed after it returns?